langwatch/src/components/TraceToPlaygroundLink.tsx

```diff
@@ -0,0 +1,86 @@
+import { Link } from "@chakra-ui/next-js";
+import { Button, Tooltip } from "@chakra-ui/react";
+import { projectRoutes } from "~/utils/routes";
+import { Image as ImageIcon } from "react-feather";
+import React from "react";
+/**
+ * Compose the link to the playground with the given project slug, trace ID, and
+ * span ID.
+ *
+ * The playground link is composed of the project slug, trace ID, and span ID in
+ * the following format:
+ *  /[projectSlug]/playground?traceId=[traceId]&span=[spanId]
+ *
+ * @param projectSlug - The project slug to link to.
+ * @param traceId - The trace ID to load in the playground.
+ * @param spanId - The span ID to load in the playground.
+ * @returns The link to the playground with the given trace and span.
+ */
+function useLinkHref(projectSlug: string, traceId: string, spanId: string) {
+  return React.useMemo(() => {
+    const queryString = new URLSearchParams({
+      traceId,
+      span: spanId,
+    }).toString();
+
+    const path = projectRoutes.playground.path.replace(
+      "[project]",
+      projectSlug
+    );
+
+    return `${path}?${queryString}`;
+  }, [projectSlug, traceId, spanId]);
+}
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace TraceToPlaygroundLink {
+  export interface Props {
+    /**
```

Comment:
I don't understand how can this comments be useful, they say the same that their variables names, the code is pretty self explanatory without the comments